framework/assets/yii.js

```diff
@@ -273,7 +273,7 @@ window.yii = (function ($) {
                 return {};
             }
 
-            var pairs = url.substring(pos + 1).split('#')[0].split('&'),
+            var pairs = url.substring(pos + 1).split('#')[0].split('&').filter(String),
```

Comment:
If it could be easily replaced with something compatible with earlier standards, it's better to use more compatible solution. If there's no easy replacement, let it be as is.